### ISSUE TITLE
Accessibility Issue: "Sign in now" link is not focusable

### ISSUE THREAD

**Issue by mattcoy-arcticleaf** [bigcommerce/checkout-js#1412]
"Sign in now" link cannot be accessed with the keyboard via tab as it is an `<a>` tag without an href. This could cause other usability issues as well. It should be a `<button>` tag since it is not actually a link.

To replicate, go to checkout in Cornerstone and tab through the elements. You will see that the Sign in now button never receives focus!

**Comment by animesh1987** [comment]
Hey @mattcoy-arcticleaf thanks for raising this, will have a look at this in next week and get back to on this.

**Comment by bc-0dp** [comment]
@bigcommerce/team-checkout was this resolved?

**Comment by mattcoy-arcticleaf** [comment]
@bc-0dp , nope, this was not resolved.

**Comment by bc-0dp** [comment]
@mattcoy-arcticleaf if you want you are very welcome to create a PR. I've made a draft one for review and consideration.
